cocotb/share/makefiles/simulators/Makefile.questa

```diff
@@ -114,25 +114,6 @@ else
 	@echo "quit" >> $@
 endif
 
-ifeq ($(OS),Msys)
-
-# Windows allows the situation where the libstc++ used at link time as
-# specified by -L can be different to the one that is used at runtime which
-# comes from the first libstdc++ that it finds in the path. As such
-# we use the mingw lib used at build time and put this at the start of the path
-# before running
-
-MINGW_BIN_DIR = $(shell dirname $(shell :; command -v gcc))
-
-OLD_PATH := $(shell echo "$(PATH)" | sed 's/(/\\(/g' | sed 's/)/\\)/g' | sed 's/ /\\ /g')
-
-LIB_LOAD := PATH=$(MINGW_BIN_DIR):$(OLD_PATH):$(LIB_DIR)
-NEW_PYTHONPATH := $(shell echo "$(PYTHONPATH)" | sed -e 's/\\/\//g' -e 's/\([a-zA-Z]\):\//\/\1\//g' -e 's/;/:/g')
-else
-LIB_LOAD :=
-NEW_PYTHONPATH := $(PYTHONPATH)
-endif
```

Comment:
How do we know we no longer need this?

Reply:
Do you ask about `MINGW_BIN_DIR` ? The only argument I have is that I spend quite a bit on time on this tested with multiple configurations and never seen this would be needed. 
I also do not think we need to fix here some strange user environment (that is very easy on Windows).  Our compilation method has changed.

Reply:
If you ask about `NEW_PYTHONPATH` not  need all this if `PYTHONHOME` is set.

Reply:
`MINGW_BIN_DIR` was my concern, I'm not worried about whatever garbage is going on in PYTHON_PATH here.

-